### PR TITLE
[glidejs__glide] Refactor typings, add components, etc

### PR DESCRIPTION
**Pull request by martin-badin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test glidejs__glide`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/glidejs/glide>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by martin-badin** [comment]
@peterblazejewicz Could you have a time and look at this one?

File: types/glidejs__glide/glidejs__glide-modular-tests.ts [hunk 1]
```diff
@@ -6,97 +6,12 @@ import Glide, {
     Images,
     Keyboard,
     Swipe,
-    Component,
-    ComponentFunction,
-    Properties,
-    TransformerFunction,
-    EventsBus,
-} from 'glidejs__glide/dist/glide.modular.esm';
+} from '@glidejs/glide/dist/glide.modular.esm';
```
Comment (peterblazejewicz) [inline types/glidejs__glide/glidejs__glide-modular-tests.ts:9]: 👍🏻

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by martin-badin** [comment]
Ready to merge

**Pull request merged**